dodo.py

```diff
@@ -0,0 +1,400 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import os
+import subprocess
+from pathlib import Path
+from subprocess import PIPE
+
+import doit
+from doit import task_params
+from doit.tools import CmdAction
+from tests.images_hierarchy import get_test_dirs
+
+# global doit config
+DOIT_CONFIG = {"verbosity": 2, "default_tasks": ["build_docs"]}
```

Comment:
I'm not sure we should have build_docs as a default task.

Reply:
I haven't found a way to show the target list by default :(

Reply:
I tried to call doit (from doit) with `list` command, but it doesn't work, because the database is locked.
If you don't know how to solve this, just press "Resolve", because this is not a big deal.

Reply:
I had placed the `build_docs` as a placeholder - while I found a good default.  Replaced now